packages/babel-plugin/src/css-prop/__tests__/jsx-pragma.test.tsx

```diff
@@ -44,11 +44,11 @@ describe('local jsx namespace', () => {
     expect(actual).toMatchInlineSnapshot(`
       "/* File generated by @compiled/babel-plugin v0.0.0 */
 
-      import { ax, ix, CC, CS } from \\"@compiled/react/runtime\\";
-
       /** @jsxRuntime classic */
 
       /** @jsx jsx */
+
+      import { ax, ix, CC, CS } from \\"@compiled/react/runtime\\";
```

Comment:
I understand why you are[ recommending](https://github.com/atlassian-labs/compiled/pull/1061#discussion_r794093223) to insert everything after the original `@compiled/react` import as the safest option. This case is an example; ideally we want to keep comments in their related/closest node. 

However, I don't think that we can rely on` @compiled/react` import being always available. I am not very familiar with jsx-pragma, but it seems that `@compiled/react` import is not required when using `/** @jsxImportSource @compiled/react */` ( eg https://compiledcssinjs.com/docs/api-css-prop).

We could insert them after the first import, but this won't work if there are not import declaration within the file.
Hence, I think moving the leading comments back up regardless seems to be the best option. Open to other suggestions though